core/dbt/logger.py

```diff
@@ -46,170 +34,381 @@ def notice(self, msg, *args, **kwargs):
 
 colorama.init(wrap=colorama_wrap)
 
-# create a global console logger for dbt
-stdout_handler = logging.StreamHandler(colorama_stdout)
-stdout_handler.setFormatter(logging.Formatter('%(message)s'))
-stdout_handler.setLevel(NOTICE)
-
-stderr_handler = logging.StreamHandler(sys.stderr)
-stderr_handler.setFormatter(logging.Formatter('%(message)s'))
-stderr_handler.setLevel(WARNING)
-
-
-logger = logging.getLogger('dbt')
-logger.addHandler(stdout_handler)
-logger.setLevel(DEBUG)
-logging.getLogger().setLevel(CRITICAL)
-
-# Quiet these down in the logs
-logging.getLogger('botocore').setLevel(INFO)
-logging.getLogger('requests').setLevel(INFO)
-logging.getLogger('urllib3').setLevel(INFO)
-logging.getLogger('google').setLevel(INFO)
-logging.getLogger('snowflake.connector').setLevel(INFO)
-logging.getLogger('parsedatetime').setLevel(INFO)
-# we never want to seek werkzeug logs
-logging.getLogger('werkzeug').setLevel(CRITICAL)
-
-# provide this for the cache.
-CACHE_LOGGER = logging.getLogger('dbt.cache')
-# add a dummy handler to avoid `No handlers could be found for logger`
-nothing_handler = logging.StreamHandler()
-nothing_handler.setLevel(CRITICAL)
-CACHE_LOGGER.addHandler(nothing_handler)
-# provide this for RPC connection logging
-RPC_LOGGER = logging.getLogger('dbt.rpc')
-
-
-# Redirect warnings through our logging setup
-# They will be logged to a file below
-logging.captureWarnings(True)
-warnings.filterwarnings("ignore", category=ResourceWarning,
-                        message="unclosed.*<socket.socket.*>")
 
-initialized = False
+STDOUT_LOG_FORMAT = '{record.message}'
+# TODO: can we change the time to just "{record.time:%Y-%m-%d %H:%M:%S.%f%z}"?
```

Comment:
The first line is really complicated and silly looking (`'{record.time:%Y-%m-%d %H:%M:%S%z},{record.time.microsecond:03}'` -> `2019-08-30 11:54:24,853`) to try and exactly copy logging's default pythong `logging` format. I'd prefer the logbook default time format (`'{record.time:%Y-%m-%d %H:%M:%S.f%z}` -> `2019-08-30 11:54:24.853978`), as it's both more precise and not comma-separated. But changing it will break anyone using regex on dbt debug output, so I wanted to leave it alone to start.

Reply:
that makes a lot of sense! Let's leave it as is for now.

If we did want to break this, now would be the time to do it though!!